MetaMask/metamask-mobile
# test: skip onramp onboarding for returning user

## **Description**

This tests that a returning user in the buy flow _does not_ see the onboarding flow again, or the steps to select region and payment method. It should skip to the build quote screen. 


## **Related issues**

https://consensyssoftware.atlassian.net/browse/RAMPS-1871

## **Manual testing steps**

All Onramp tests are passing

```
 PASS  e2e/specs/ramps/onramp.spec.js (52.479 s)
  smokeAssets Buy Crypto
    ✓ should select Region and Payment Method to see the Build Quote screen (16291 ms)
    ✓ should skip to the Build Quote screen for returning user (14077 ms)

Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        52.55 s, estimated 55 s
```

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
